Source/Events.Store.MongoDB/Streams/EventFetchers.cs

```diff
@@ -87,21 +87,23 @@ public async Task<ICanFetchEventTypesFromPartitionedStream> GetPartitionedTypeFe
             return await GetPartitionedFetcherFor(scopeId, streamDefinition, cancellationToken).ConfigureAwait(false) as ICanFetchEventTypesFromPartitionedStream;
         }
 
-        async Task<StreamFetcher<MongoDB.Events.Event>> CreateStreamFetcherForEventLog(ScopeId scopeId, CancellationToken cancellationToken) =>
-            new StreamFetcher<MongoDB.Events.Event>(
+        async Task<StreamFetcher<Events.Event>> CreateStreamFetcherForEventLog(ScopeId scopeId, CancellationToken cancellationToken) =>
```

Comment:
Is the Codacy thing complaining on these qualified types? The reason I did it like this in the Events.Store.MongoDB project is because there are two separate representations of the event types; the Runtime representation in Events.Store and the MongoDB representation in Events.Store.MongoDB. And to make it clear which representation it's working on I partially qualified the types all over the place. Another way we could deal with this is to have using aliases for either the Runtime or the MongoDB types. I don't know what you think we should do?

Reply:
I don't think so, just my VScode passive aggressively suggesting it. But I don't really mind either way. I've reverted both files.